cpp/src/gandiva/precompiled/CMakeLists.txt

```diff
@@ -31,14 +31,13 @@ set(PRECOMPILED_SRCS
     ../../arrow/util/basic_decimal.cc)
 
 if(MSVC)
-  # clang pretends to be a particular version of MSVC. Version 1900 is
-  # Visual Studio 2015, 191[0-9] is Visual Studio 2017, and the standard
-  # library uses C++14 features, so we have to use that -std version to
-  # get the IR compilation to work
+  # clang pretends to be a particular version of MSVC. 191[0-9] is
+  # Visual Studio 2017, and the standard library uses C++14 features,
+  # so we have to use that -std version to get the IR compilation to work
   if(MSVC_VERSION MATCHES "^191[0-9]$")
```

Comment:
Does this mean that we support all MSVC versions starting from VS2017?

Reply:
As much as possible, yes.